releases.yml

```diff
@@ -1,5 +1,48 @@
 releases:
 
+  ec.1:
+    assembly:
+      basis:
+        brew_event: 55544508
+        reference_releases:
+          aarch64: 4.16.0-0.nightly-arm64-2024-01-09-085007
+          ppc64le: 4.16.0-0.nightly-ppc64le-2024-01-09-085007
+          s390x: 4.16.0-0.nightly-s390x-2024-01-09-085009
+          x86_64: 4.16.0-0.nightly-2024-01-09-085011
+      group:
+        advisories:
+          extras: 125774
+          image: 125775
+          metadata: 125776
+          microshift: 125777
+          prerelease: -1
```

Comment:
```diff
-          prerelease: -1
+          prerelease: 125778
```

Reply:
Right, but how come gen-assembly omitted this?